content/client-lib-development-guide/features.textile

```diff
@@ -232,6 +232,9 @@ h3(#rest-auth). Auth
 ** @(RSA16b)@ If the library is provided with a token without the corresponding @TokenDetails@, then this holds a @TokenDetails@ instance in which only the @token@ attribute is populated with that token string
 ** @(RSA16c)@ Is set with the current token (if applicable) on instantiation and each time it is replaced, whether the result of an explicit @Auth#authorize@ operation, or a library-initiated renewal resulting from expiry or a token error response
 ** @(RSA16d)@ Is empty if there is no current token, including after a previous token has been determined to be invalid or expired, or if the library is using basic auth
+* @(RSA17)@ @Auth#revokeToken@ function:
```

Comment:
Why not `revokeTokens` to reflect (i) what it does, and (ii) the REST endpoint name ?

Reply:
Changed method name ea9c6dce7fa0f04a08bf3494a73236c9813fd2f1 and added explanation and REST endpoint name 3819be5d3eeca7380d45921323ae953837fc0bcc

Reply:
@paddybyers If this conversation needs to continue, then that will need to happen against the new pull request that we will open against https://github.com/ably/specification/issues/10. Please resolve this conversation _here_ once you have re-raised your concern _there_, or if you are already happy that this matter is resolved. Thanks.